### PR TITLE
Add support for flatpak

### PR DESCRIPTION
**Pull request by emiljoha**

### CODE REVIEW

**Comment by emiljoha** [comment]
Issues seem to stem from that we get NULL back from wl_display_connect in wl-copy.c. Somehow we are not getting access to the wayland server.

```
    struct wl_display *wl_display = wl_display_connect(NULL);
    if (wl_display == NULL) {
        bail("Failed to connect to a Wayland server");
    }
```

**Comment by emiljoha** [comment]
We are also now finding any config file but that part should be easier to fix.
```bash
flatpak run se.emijoh.mpw
Warning: Failed to open config file
Full Name: fsdfs
Site Name: fsdf
Password: 
Failed to connect to a Wayland server
Warning: Could not find a copy/paste mechanism for your system.
[ ╚☻╯☎ ]: FongVufsQahe7#
```

**Comment by emiljoha** [comment]
```wl-clipboard``` generally seem to be happy with its dependencies during build.

Part of output during build:
```
Run-time dependency wayland-client found: YES 1.18.0
Program wayland-scanner found: YES
Using 'PKG_CONFIG_PATH' from environment with value: '/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig'
Run-time dependency wayland-protocols found: YES 1.20
```
Header <sys/syscall.h> has symbol "SYS_memfd_create" : YES 
Header <sys/mman.h> has symbol "SHM_ANON" : NO

File: se.emijoh.mpw.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        path: mpwcli.py
+  - python3-pyperclip.json
+  - python3-mpwalg.json
+  - name: wl-clipboard
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://github.com/bugaevc/wl-clipboard/archive/v2.0.0.tar.gz
+        sha256: 2c42f182432adabe56da0f1144d5fcc40b7aae3d8e14d2bc4dc4c3f91b51808d
+finish-args:
```
Comment (wjt) [inline se.emijoh.mpw.yml:22]: Add `--socket=wayland`

Comment (emiljoha) [reply se.emijoh.mpw.yml:22]: Thanks! 

I have tried adding the --socker=wayland but flatpak never added the permission as i did not add that extra `-` before each element in the yml list :facepalm: 

Now I can write to the cliboard!

**Pull request merged**